source/assets/stylesheets/scss/_downloads.scss

```diff
@@ -787,3 +789,33 @@ body.download {
   background:url(../images/dld-k8s.svg) no-repeat 5px 10px ;
   background-size:20px;
 }
+
+.steps-section{
```

Comment:
@Chandu199 Can you format this (using the IDE maybe)? So that it becomes:

```
.steps-section {
```

Just adding an extra space before the brace. This is in many places.